datasetviewer/tests/file_loader_presenter_test.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+
+import mock
+
+from enum import Enum
+
+from datasetviewer.fileloader.FileLoaderPresenter import FileLoaderPresenter
+from datasetviewer.fileloader.interfaces.FileLoaderView import FileLoaderView
+from datasetviewer.fileloader.Command import Command
+from datasetviewer.mainview.MainViewPresenter import MainViewPresenter
+
+import xarray as xr
+
+class FileLoaderPresenterTest(unittest.TestCase):
+
+    def setUp(self):
+
+        self.main_presenter = mock.create_autospec(MainViewPresenter)
+        self.main_presenter.set_data = mock.MagicMock()
+        self.view = mock.create_autospec(FileLoaderView)
+
+        self.dummy_data = xr.Dataset()
+        self.fake_file_path = "filepath"
+
+        self.view.get_selected_file_path = mock.MagicMock(side_effect=lambda: self.fake_file_path)
+
+        self.fl_presenter = FileLoaderPresenter(self.view)
+        self.fl_presenter.register_master(self.main_presenter)
+
+    def test_presenter_throws_if_view_none(self):
+
+        with self.assertRaises(ValueError):
+            FileLoaderPresenter(None)
+
+    def test_register_master(self):
+
+        fl_presenter = FileLoaderPresenter(self.view)
+
+        main_presenter = mock.create_autospec(MainViewPresenter)
+        fl_presenter.register_master(main_presenter)
+
```

Comment:
I understand what these tests are doing, but others might not. Where you are testing that functions have been called. Can you add a comment line to explain why they should be called. i.e. 
```python
# Calling register_master on sub-presenter should also ensure a two-way link by ensuring that the main presenter is subscribed to a sub-presenter. We are testing that the two-way link works.
```